lib/Internal/Request.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace Aerys;
+namespace Aerys\Internal;
```

Comment:
This shouldn't be Internal. It's the raw request information, necessary for middlewares to mutate or access client.

Reply:
Middlewares will use `Aerys\Request`.

Reply:
`Aerys\Request` will have a method that will return client information.